src/app.ts

```diff
@@ -46,8 +46,7 @@ const processPushEvent = async (payload: PushEvent, context: Context<'push'>) =>
       log.info(`See: https://github.com/${repository.owner}/${repository.repo}/pull/${pullRequestNumber}`)
     }
   } catch (e: unknown) {
-    log.error(`Failed to process commit '${payload.after}' with error:`)
-    log.error(e as never)
+    throw new Error(`Failed to process commit '${payload.after}' with error: ${e}`)
```

Comment:
This will kill the app afaict 😬 Not something we'd want - it's fine that a repository fails, but we shouldn't prevent the app from running on all other events coming in after a failure



```suggestion
    log.error(`Failed to process commit '${payload.after}' with error:`)
    log.error(e as never)
```

Reply:
Hmm, are you sure? I've just tested it locally and it seems to be not the case. Error is thrown and app keeps going and handling other requests.

Reply:
I have tested locally and it keeps logging the error with the event and goes on.